pkg/controller/common/util.go

```diff
@@ -1194,6 +1197,9 @@ func CreatePvcInStorageClass(name, ns string, storageClassName *string, annotati
 		},
 	}
 	pvc.Status.Capacity = pvc.Spec.Resources.Requests.DeepCopy()
+	if pvc.Status.Phase == corev1.ClaimBound {
+		pvc.Spec.VolumeName = "pv-" + string(pvc.UID)
+	}
```

Comment:
This function is only used in unit tests.  Weird that it is here, right?

Reply:
Yeah that is weird, I guess its in this package because a bunch of other packages use it. But I think it could be in util_test.go if only tests use it.

Reply:
unfortunately it doesn't appear that the function will be importable from other packages if we do that though

Reply:
That is probably why the function is in here.